pkg/commandhandler/cmd.go

```diff
@@ -70,10 +71,13 @@ func execCmdInDirWithAction(cmd, workingDir, logDir string, waitForCompletion bo
 			stdOutFile.Close()
 		}
 
-		stdErrFile, err3 := os.OpenFile(errFileName, os.O_RDONLY, constants.FilePermissions_UserOnly_ReadWrite)
+		stdErrContents, err3 := os.ReadFile(errFileName)
 		if err3 == nil {
-			el.InfoFromStream("stderr:", stdErrFile)
-			stdErrFile.Close()
+			el.Info("stderr: %s", stdErrContents)
+
+			if execErr != nil {
+				return exitCode, errors.Wrap(execErr, string(stdErrContents))
```

Comment:
We probably shouldn't fail the command if we can't read the standard error, right?